web/src/layout/main.js

```diff
@@ -98,8 +104,28 @@ const Main = ({
                 ]}
                 value={electricityMixMode}
               />
+              <Toggle
+                onChange={value => dispatchApplication('timescale', value)}
+                options={[TIMESCALE.MONTHLY, TIMESCALE.LIVE].map((k) => {
+                  if (k === TIMESCALE.MONTHLY) {
+                    return { value: k, label: 'monthly historical' };
+                  }
+                  if (k === TIMESCALE.LIVE) {
+                    return { value: k, label: 'live' };
+                  }
+                  return { value: 'unknown', label: 'unknown' };
+                })}
```

Comment:
Why not simply `options={{ value: TIMESCALE.MONTHLY, label 'monthly historical' }, { value: TIMESCALE.LIVE, label: 'live }}`?